mapsims/tests/data/make_benchmark.py

```diff
@@ -0,0 +1,85 @@
+import sys
+sys.path.append("../..")
+
+import numpy as np
+import SO_Noise_Calculator_Public_20180822 as so_n
+import healpy as hp
+import matplotlib.pyplot as plt
+
+nside = 16
+lmax = 500
+seed = 1234
+channel = 2
+
+nh_LA = hp.ud_grade(
+    hp.read_map("../../data/total_hits_LA_classical.fits.gz", verbose=False),
+    nside_out=nside,
+)
+nh_LA /= np.amax(nh_LA)
+fsky_LA = np.mean(nh_LA)
+ls_LA, Nl_LA_T, Nl_LA_P, _ = so_n.Simons_Observatory_V3_LA_noise(1, fsky_LA, lmax, 1)
+# Extending to l=0
+Nl_LA_Tb = np.zeros([len(Nl_LA_T), len(ls_LA) + 2])
+Nl_LA_Tb[:, 2:] = Nl_LA_T
+Nl_LA_Tb[:, :2] = Nl_LA_T[:, 0][:, None]
+Nl_LA_T = Nl_LA_Tb
+Nl_LA_Pb = np.zeros([len(Nl_LA_P), len(ls_LA) + 2])
+Nl_LA_Pb[:, 2:] = Nl_LA_P
+Nl_LA_Pb[:, :2] = Nl_LA_P[:, 0][:, None]
+Nl_LA_P = Nl_LA_Pb
+ls_LA = np.arange(len(ls_LA) + 2)
+
+nh_SA = hp.ud_grade(
+    hp.read_map("../../data/total_hits_SA_classical.fits.gz", verbose=False),
+    nside_out=nside,
+)
+nh_SA /= np.amax(nh_SA)
+fsky_SA = np.mean(nh_SA)
+ls_SA, Nl_SA_P, _ = so_n.Simons_Observatory_V3_SA_noise(1, 1, 1., fsky_SA, lmax, 1)
+# Extending to l=0
+Nl_SA_Pb = np.zeros([len(Nl_SA_P), len(ls_SA) + 2])
+Nl_SA_Pb[:, 2:] = Nl_SA_P
+Nl_SA_Pb[:, :2] = Nl_SA_P[:, 0][:, None]
```

Comment:
@damonge what about setting the power spectrum value at ell=1 and ell=0 to zero instead of setting them equal to the value of noise at ell=2?

Reply:
@zonca That sounds good to me. We shouldn't use those multipoles in any of our forecasts anyway.

Reply:
ok, thanks, implemented in a3c555c010d6318cad025c97614cf6037b7aa6a0